justfile

```diff
@@ -46,3 +46,7 @@ mount-docker-backup *args:
 
 restore *args:
     docker compose exec app_db scripts/restore {{args}}
+
+test:
```

Comment:
pytest is frequently used with args (e.g. -v, -x, -s, -vvs, -vvsx)

I suggest to add `*args` to the command

Reply:
Done :-)